reporting/src/main/resources/application.properties

```diff
@@ -1,9 +1,4 @@
 kafka.bootstrap.address=broker:29092
 reporting.subscriber=reportingSubscriber
 spring.data.mongodb.uri=mongodb://root:example@mongo:27017/admin
-#spring.data.mongodb.uri=mongodb://localhost:27017/admin
-managementms.hostname=http://management
-managementms.port=8080
-#managementms.hostname=http://localhost
-#managementms.port=8081
-#server.port=8084
```

Comment:
Should we be removing these entries? How do we default to config entries in case no ENV variable specified?

Reply:
this came into picture in today's standup. will fix it